test/no_pmix/cart_nopmix_multictx_one_node.py

```diff
@@ -73,15 +73,16 @@ def test_cart_no_pmix(self):
         self.utils.print("\nTest cmd : %s\n" % cmd)
 
         test_env = self.pass_env
-        p = subprocess.Popen([cmd], env=test_env, stdout=subprocess.PIPE)
+        #p = subprocess.Popen([cmd], env=test_env, stdout=subprocess.PIPE)
```

Comment:
Is this temporary?  Maybe a comment with a ticket # would be good here.

Reply:
Actually this one we will likely be removing. This test violates this check rule by lying about uris so it ends up sending to wrong targets

Reply:
seems like a good test that it returns the error code, no?